contracts/05-Withdraw/Withdraw.sol

```diff
@@ -34,23 +34,26 @@ contract Withdraw is Access, IWithdraw {
      * @notice Proposes a withdraw poll with the default withdrawPollDuration.
      * @param _amount Size of the proposed withdrawal.
      * @param _beneficiary Beneficiary of the reward.
+     * @param _unlockDate Date beyond which it will be possible to withdraw
      */
-    function proposeWithdraw(uint256 _amount, address _beneficiary) external override onlyOwner {
+    function proposeWithdraw(uint256 _amount, address _beneficiary, uint256 _unlockDate) external override onlyOwner {
         require(_amount != 0, 'NOT_VALID');
 
         _createWithdrawPoll(
             _amount,
             LibWithdrawPollStorage.withdrawStorage().proposeWithdrawPollDuration,
-            _beneficiary
+            _beneficiary,
+            _unlockDate
         );
     }
 
     /**
      * @notice Proposes a withdraw poll with the default withdrawPollDuration in bulk.
      * @param _amounts Sizes of the proposed withdrawal.
      * @param _beneficiaries Beneficiaries of the reward.
+     * @param _unlockDate Date beyond which it will be possible to withdraw
      */
-    function proposeBulkWithdraw(uint256[] memory _amounts, address[] memory _beneficiaries)
+    function proposeBulkWithdraw(uint256[] memory _amounts, address[] memory _beneficiaries, uint256 _unlockDate)
```

Comment:
@valeriagrazzini If we'd make _unlockData and array of dates, its a bit more flexible since we will be able to use different unlockDates per withdrawal. Do you agree?

Reply:
Yes that was one of my doubts..of course we can do it, but then, when I call function _withdrawPollFinalize(), will I need to specify the wpool index to get the value of unlockDate?

Reply:
Thats not relevant to this change right? What you describe also applies to the (single) proposeWithdraw method. 

What you describe is solved in the proxy method invokes the private _withdrawPollFinalize() method. See: 
https://github.com/thxprotocol/modules-solidity/blob/main/contracts/05-Withdraw/WithdrawPollProxy.sol#L40-L47

This proxy exists to be able to use a similar implementation for these two scenarios:
1. Let the user call this method himself (and pay for the gas himself)
2. User our relayer to call the method (admin pas for the gas)

When the user finalizes a withdrawal the withdrawalID is passed to that proxy method, so as soon as _withdrawPollFinalize() is invoked, it will be clear what withdrawal (and which unlockDate from storage) it needs to use.